### PR TITLE
Add i18n context for resolving invalid blocks

### PR DESCRIPTION
**Pull request by brandonpayton**
## Description
This is a simple PR to add i18n context for "Resolve" and "Resolve Block" strings to resolve translation concerns from [this comment](https://github.com/WordPress/gutenberg/pull/8274#issuecomment-431014989).

@gAllegr, does this address your issue with the strings?

## Screenshots

screen shot 2018-10-19 at 12 18 12 am
screen shot 2018-10-19 at 12 18 20 am

### CODE REVIEW

**Comment by gAllegr** [comment]
Yes, thank you, these screenshots are much more explicative.
Even better would be add a specific comment for the two strings, something like the following screenshot. While translating it's not obvious where strings are located.
Maybe _Button to fix content_ for _Resolve_ string and _Window title to fix content_ for the _Resolve block_ one?

comment

**Comment by brandonpayton** [comment]
Thanks for the additional suggestions, @gAllegr. This is good to be learning.

I made an update based on your comments. How does the latest look?

**Review by mcsf — APPROVED** [review]
Nitpick: is the line break needed at all?

```js
// translators: Dialog title to fix block content
title={ __( 'Resolve Block' ) }
```

other than that question: good to merge, the notes and syntactic case are good.

**Comment by aduth** [comment]
>Nitpick: is the line break needed at all?

The parser is a little picky, and while the comment doesn't need to strictly be a leading comment of the translation call, it _does_ need to be a leading comment of one of its parents in the AST hierarchy. This might actually work here as the leading comment of the attribute assignment, but it's worth verifying.

tl;dr The parser doesn't strictly consider "previous line", it considers leading comments of its AST ancestry which occur on the current or previous line.

See https://github.com/WordPress/gutenberg/blob/a54896121a115ad143fd9b6ac331ca8eedf212af/packages/babel-plugin-makepot/src/index.js#L109-L157

**Comment by gAllegr** [comment]
> Thanks for the additional suggestions, @gAllegr. This is good to be learning.
> 
> I made an update based on your comments. How does the latest look?

Looks ok to me
Generally speaking, while coding and preparing for internationalization, if you found something that might generate confusion it is great to add a comment for translators

**Comment by aduth** [comment]
>Nitpick: is the line break needed at all?

Based on #11494 / #11514, it appears it is needed (as merged here should be fine).
